opcua/common/connection.py

```diff
@@ -290,7 +290,6 @@ def receive_from_header_and_body(self, header, body):
             return msg
         elif header.MessageType == ua.MessageType.Error:
             msg = struct_from_binary(ua.ErrorMessage, body)
-            logger.warning("Received an error: %s", msg)
```

Comment:
why did you remove logging here? I know these libraries logs too much but errors should happen that often...

Reply:
I thought it'd be a duplicate, but I've restored it.